### PR TITLE
[wave2-1035] clean pr for 1035

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

**Comment by dpfitzsi** [comment]
Already reviewed in previous PR.

Mike can you resolve these CSS conflicts before we merge?

File: sites/all/themes/ucgy_theme/resources/scss/bricks_custom.scss [hunk 1]
```diff
@@ -383,11 +383,11 @@ a.compare-cart {
     .results {
 
       // Program requirements spinner
-      // There are other 'loading spinners' we need to wrangle together
       .loading {
         text-align: center;
-        height: 45px + 15px; // + bottom spacing
-        background-position: top center;
+        margin-bottom: 20px;
```
Comment (dpfitzsi) [inline sites/all/themes/ucgy_theme/resources/scss/bricks_custom.scss]: Looks like new loading css is being overridden in this branch and the new ajax-loader.gif is being delete.

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/ucgy_core_variable_table_form.info [hunk 2]
```diff
@@ -1,15 +1,6 @@
-core = "7.x"
-description = "A variable table form for getting/setting variables"
-name = "UCGY Variable Table Module"
-features[ckeditor_profile][] = Advanced
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/ucgy_core_variable_table_form.info]: Is there a reason that all of the dependencies have been deleted from this module?

Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_variable_table_form/ucgy_core_variable_table_form.info]: we didn't need these dependencies for the Variable Table Form, they shouldn't have been there in the first place.

File: sites/all/themes/ucgy_theme/resources/scss/bricks_custom.scss (hunk 1 above)
Comment (mhughes2012) [inline sites/all/themes/ucgy_theme/resources/scss/bricks_custom.scss]: moved to put the theme from fgs branch to this branch

**Comment by mhughes2012** [comment]
merged theme from fgs to this branch, added missing updated department view

**Pull request merged**